src/elisp/treemacs-mouse-interface.el

```diff
@@ -48,6 +52,17 @@ Must be bound to a mouse click, or EVENT will not be supplied."
        :no-error            t))
     (treemacs--evade-image)))
 
+(defun treemacs-downleftclick-action (event)
```

Comment:
AFAICT this method is largely a duplication of the leftclick-action above, only used for saving the clicked node. If so we should combine them.